css/src/utils/border.css

```diff
@@ -38,6 +38,56 @@
   border-left: 0 !important;
 }
 
+/* Border color utilities */
+.border-primary {
+  border-color: var(--primary) !important;
+}
+
+.border-secondary {
+  border-color: var(--secondary) !important;
+}
+
+.border-success {
+  border-color: var(--success) !important;
+}
+
+.border-warning {
+  border-color: var(--warning) !important;
+}
+
+.border-alert {
+  border-color: var(--alert) !important;
+}
+
+.border-accent1 {
+  border-color: var(--accent1) !important;
+}
+
+.border-accent2 {
+  border-color: var(--accent2) !important;
+}
+
+.border-accent3 {
+  border-color: var(--accent3) !important;
+}
+
+.border-accent4 {
+  border-color: var(--accent4) !important;
+}
```

Comment:
Can we consider to update this classes with `border-accent-1`, `border-accent-2`, etc.